jquery.flot.navigate.js

```diff
@@ -82,6 +82,18 @@ can set the default in the options.
             active: false,
             cursor: "move",
             frameRate: 60
+        },
+        xaxis: {
```

Comment:
Could you please move the comments in the documentation section on top of the file?